test/Atmos/EDMF/stable_bl_edmf.jl

```diff
@@ -198,7 +200,7 @@ function main(::Type{FT}) where {FT}
     grid = driver_config.grid
 
     # state_types = (Prognostic(), Auxiliary(), GradientFlux())
-    state_types = (Prognostic(), Auxiliary())
+    state_types = (Prognostic(), Auxiliary(), GradientFlux())
```

Comment:
if we are exporting GradientFlux and entrainment and detrainment are in the output could we compute all the needed values offline ?

Reply:
Well yes but I would have to rewrite the EDMF offline, I don't see the point if I can just get the output here. The same is applicable to entrainment and detrainment.

Reply:
The ideal think would be if I could get the `atmos_source!` variables